android/res/xml/locales_config.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<locale-config xmlns:android="http://schemas.android.com/apk/res/android">
+    <locale android:name="en" />
+    <locale android:name="ar" />
+    <locale android:name="be" />
+    <locale android:name="bg" />
+    <locale android:name="ca" />
+    <locale android:name="cs" />
+    <locale android:name="da" />
+    <locale android:name="de" />
+    <locale android:name="el" />
+    <locale android:name="en-rGB" />
+    <locale android:name="es" />
+    <locale android:name="es-rMX" />
+    <locale android:name="eu" />
+    <locale android:name="fa" />
+    <locale android:name="fi" />
+    <locale android:name="fr" />
+    <locale android:name="fr-rCA" />
+    <locale android:name="hu" />
+    <locale android:name="in" />
+    <locale android:name="it" />
+    <locale android:name="iw" />
+    <locale android:name="ja" />
+    <locale android:name="ko" />
+    <locale android:name="mr" />
+    <locale android:name="nb" />
+    <locale android:name="nl" />
+    <locale android:name="pl" />
+    <locale android:name="pt" />
+    <locale android:name="pt-rBR" />
+    <locale android:name="ro" />
+    <locale android:name="ru" />
+    <locale android:name="sk" />
+    <locale android:name="sv" />
+    <locale android:name="sw" />
+    <locale android:name="th" />
+    <locale android:name="tr" />
+    <locale android:name="uk" />
+    <locale android:name="vi" />
+    <locale android:name="zh" />
+    <locale android:name="zh-rHK" />
+    <locale android:name="zh-rMO" />
+    <locale android:name="zh-rTW" />
```

Comment:
Are you sure about these? In the example they use zh-Hant and zh-Hans. Can you please test if it works properly?